Project/BookDetail/Comments/CommentViewController.swift

```diff
@@ -27,16 +30,46 @@ class CommentViewController: UIViewController , UITableViewDelegate, UITableView
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        _view.commentTable.delegate = self
+        _view.commentTable.rowHeight = 120
+        _view.commentTable.dataSource = self
         _view.commentTable.register(cell: CommentView.self)
-        //view.backgroundColor = .backgroundColor
+        _view.commentTable.separatorStyle = .none
```

Comment:
You can send all of this `_view.commentTable` lines to a new function called, I don't know, `func setupCommentTable()`?